digits/dataset/images/classification/views.py

```diff
@@ -167,6 +203,37 @@ def from_files(job, form):
     if not image_folder:
         image_folder = None
 
+    augmentation = {
+    }
+
+    if form.has_augmentation_contrast.data:
+        augmentation['contrast'] = {
+            'strength': form.augmentation_contrast_strength.data,
```

Comment:
I might be missing something but I think this field does not exist? Actually when trying to create a dataset from text files I get:
```
  File "/fast-scratch/gheinrich/ws/digits/digits/dataset/images/classification/views.py", line 211, in from_files
    'strength': form.augmentation_contrast_strength.data,
AttributeError: 'ImageClassificationDatasetForm' object has no attribute 'augmentation_contrast_strength'
```
Besides this code looks similar to lines 91 to 192 but is not exactly the same (I think the correct version is in the lines 91 to 192). Can you move this to a function and remove the duplicate code?